packages/dapp/src/hooks/useStyle.ts

```diff
@@ -0,0 +1,34 @@
+import { useState, useMemo, useCallback } from 'react';
+import { grommet } from 'grommet/themes';
+import { generate } from 'grommet/themes/base';
+import { deepMerge } from 'grommet/utils';
+import type { ThemeType } from 'grommet/themes';
+
+export enum ThemeMode {
+  light = 'light',
+  dark = 'dark'
+}
+
+export type UseStyleHook = [
+  theme: ThemeType,
+  themeMode: ThemeMode,
+  setThemeMode: Function
+];
+
+const initalThemeMode =  localStorage.getItem("themeMode") as ThemeMode;
```

Comment:
it is better to implement an internal API for working with such storage variables.
Please create a task in the backlog for this with high priority (because this is a very basic feature). We will discuss it during the next planning.

Reply:
If you resolve this comment, you must add your comment with a backlog task Id